src/lib/busy-indicator/busy-indicator.html

```diff
@@ -1,16 +1,16 @@
 <template>
   <forge-backdrop part="backdrop" fixed></forge-backdrop>
-  <aside class="forge-busy-indicator__surface" role="alertdialog" aria-modal="true" part="surface" tabindex="0">
-    <h3 class="forge-busy-indicator__title" part="title">Loading</h3>
+  <div class="forge-busy-indicator__surface" role="alertdialog" aria-modal="true" part="surface" tabindex="0">
```

Comment:
This should include `aria-labelledby` and `aria-describedby` attributes to conform to W3's example

https://www.w3.org/WAI/ARIA/apg/patterns/alertdialog/examples/alertdialog/